internal/credentials/spiffe/spiffe.go

```diff
@@ -40,27 +44,38 @@ type partialParsedSPIFFEBundleMap struct {
 // If duplicate keys are encountered in the JSON parsing, Go's default unmarshal
 // behavior occurs which causes the last processed entry to be the entry in the
 // parsed map.
-func LoadSPIFFEBundleMap(filePath string) (map[string]*spiffebundle.Bundle, error) {
+func LoadSPIFFEBundleMap(filePath string) (BundleMap, error) {
```

Comment:
This is only used from tests at this point. Do we need this anymore?

Reply:
I think so for further xds work

Reply:
I would recommend getting rid of it now and adding it back if and when required. It's only a few lines of code anyways. See: https://engdoc.corp.google.com/eng/doc/tott/episodes/737.md?cl=head

Reply:
Ack, removed

Reply:
Moved to the test file